src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -265,6 +265,7 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
             {
                 throw new Exception("Starting endpoints took longer than 2 minutes");
             }
+            await completedTask.ConfigureAwait(false);
```

Comment:
@danielmarbach yes. Without it, any exception that happens when the endpoints starts is swallowed (logged to the console but the test turns out green)